configure

```diff
@@ -6228,6 +6232,25 @@ else
 fi
 
 
+# Check whether --enable-OMR_THR_SYNCHRONIZATION_PROTOTYPES was given.
+if test "${enable_OMR_THR_SYNCHRONIZATION_PROTOTYPES+set}" = set; then :
+  enableval=$enable_OMR_THR_SYNCHRONIZATION_PROTOTYPES; if test "x${enableval}" = xyes; then :
+  OMR_THR_SYNCHRONIZATION_PROTOTYPES=1
+
+   $as_echo "#define OMR_THR_SYNCHRONIZATION_PROTOTYPES 1" >>confdefs.h
+
+else
+  OMR_THR_SYNCHRONIZATION_PROTOTYPES=0
+
+
+fi
+else
+  OMR_THR_SYNCHRONIZATION_PROTOTYPES=0
+
+
+fi
+
+
```

Comment:
I was going to complain about the gratuitous blank lines but it matches the rest of the file....